README.md

```diff
@@ -33,7 +33,7 @@ from the master branch.
 All feature growth should happen first on the master branch, but will also be
 cherry picked back to the latest yocto release branch. Security and bug fixes
 will be evaluated case by case and backported as necessary. The ambition is to
-actively maintain the four latest releases and/or two year old releases in
+actively maintain the two latest releases and/or two year old releases in
```

Comment:
> ... actively maintain the two latest releases and/or two year old releases in ...

I think here it should be `one year old releases` as the Yocto/OE release cadence is 2 releases per year (https://wiki.yoctoproject.org/wiki/Releases).

Reply:
Alright. Fixing it.